### ISSUE TITLE
Enable "go to definition" for CSS and Less

### ISSUE THREAD

**Issue by mrmckeb** [mrmckeb/typescript-plugin-css-modules#183]
This has been added for Sass (both variants) only for now.

We should now be able to rollout support for CSS using the same approach.

**Comment by mariusGundersen** [comment]
what is holding this back?

**Comment by mariusGundersen** [comment]
I tried now just enabling sourcemap in less and it seems to work. So should be easy to add this single line to the code instead of having it in the less options.

```json
    "plugins": [{
      "name": "typescript-plugin-css-modules",
      "options": {
        "classnameTransform": "camelCaseOnly",
        "goToDefinition": true,
        "rendererOptions": {
          "less": {
            "sourceMap": true
          }
        }
      }
    }]
```

**Comment by mariusGundersen** [comment]
I've tested it a bit and it seems like it should be straight forward to add this for less as well. 

The only problem is the mapping of the sourcemaps back to the original location, as pointed out here: https://github.com/less/less.js/issues/2033

That is, unless you start a nested line with & it will not go to the right line. The issue seems to be somewhere in postcss where it maps the output sourcemap from less and then picks the wrong column or something.

**Comment by mrmckeb** [comment]
Hi @mariusGundersen - just time, as I'm really the sole maintainer of this project and do all of this work in my free time. Thanks for #192!

**Comment by mrmckeb** [comment]
This is now available in `v4.2.1`:
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.1
- https://github.com/mrmckeb/typescript-plugin-css-modules/releases/tag/v4.2.0
